.eslintrc.yml

```diff
@@ -0,0 +1,31 @@
+extends: "airbnb"
+parser: "babel-eslint"
+parserOptions:
+  ecmaVersion: 6
+plugins:
+  - "react"
+rules:
```

Comment:
@osdiab I sort of want to continue the move towards zero exceptions, and only adding them when they're really necessary / sort of out of our control (e.g. the kayvee naming exception). Can you try removing all rules except the kayvee one and seeing what happens?